roles/kubernetes-apps/metrics_server/templates/metrics-server-service.yaml.j2

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: metrics-server
+  namespace: kube-system
+  labels:
+    kubernetes.io/name: "Metrics-server"
```

Comment:
lowercase M

Reply:
@woopstar 

Thanks for careful comments. 

Actually, I copy manifests from metrics server official site:

 * https://github.com/kubernetes-incubator/metrics-server/tree/master/deploy/1.8+

But should I create manifests from following kubernetes addons site?

 * https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/metrics-server

As far as I read your comments, it seems based on kubernetes addons. And I ask you.

thanks,